speech/grpc/src/test/java/com/examples/cloud/speech/StreamingRecognizeClientTest.java

```diff
@@ -52,11 +52,13 @@
  */
 @RunWith(JUnit4.class)
 public class StreamingRecognizeClientTest {
-  private TestAppender appender;
+  private Writer writer;
+  private WriterAppender appender;
 
   @Before
```

Comment:
Should this be a `@BeforeClass`, since it's adding an appender (rather than replacing it)? ie if this ones once per test, won't new appenders get added as many times as there are tests?